src/config.ts

```diff
@@ -120,20 +123,34 @@ function normalizeDest(config: SnowpackConfig) {
 
 export default function loadConfig(flags: CLIFlags) {
   const cliConfig = expandCliFlags(flags);
-  const explorerSync = cosmiconfigSync('snowpack', {
+
+  const explorerSync = cosmiconfigSync(CONFIG_NAME, {
     // only support these 3 types of config for now
     searchPlaces: ['package.json', 'snowpack.config.js', 'snowpack.config.json'],
     // don't support crawling up the folder tree:
     stopDir: path.dirname(process.cwd()),
   });
-  const result = explorerSync.search(); // search for snowpack config
 
-  // user has no config
+  let result;
+
+  // if user specified --config path, load that
+  const errors: string[] = [];
+  if (flags.config) {
+    result = explorerSync.load(flags.config);
```

Comment:
can we pass the full, absolute path to `load()` here?

```
path.resolve(flags.config)
```

That's always better when we need to report an error, in case there's a bug / unexpected behavior in our resolution logic.